apps/backend/src/app/utils/mail/mail.service.ts

```diff
@@ -0,0 +1,60 @@
+import { MailerService } from '@nestjs-modules/mailer';
+import { Injectable, Logger } from '@nestjs/common';
+import { ConfigService } from '@nestjs/config';
+import * as path from 'path';
+
+@Injectable()
+export class MailService {
+  readonly logger = new Logger(MailService.name);
+
+  constructor(
+    private mailerService: MailerService,
+    private configService: ConfigService
+  ) {}
+
+  async sendAlertMail(reason: string, description: string) {
+    const to =
+      this.configService.getOrThrow<string>('MAILING_LIST').split(',') || [];
+    const context = {
+      reason,
+      description,
+    };
+
+    const logoPath = path.resolve('apps/backend/src/assets/team_logo.png');
+
+    const attachments = [
+      {
+        filename: 'logo.png',
+        path: logoPath,
+        cid: 'logo',
+        contentType: 'image/png', // Ensure the MIME type is set correctly
+      },
+    ];
+    await this.sendMail(
+      to,
+      'Alert has been triggered',
+      'alertMail',
+      context,
+      attachments
+    );
+  }
+
+  async sendMail(
+    to: string[],
```

Comment:
I think that instead of `to`, a more descriptive name, like `recipients` or someting like that would be easier to read, and the plural would imply that an array is expected as input.

Reply:
Used the libraries domain language, but yes makes sense, I'll change it